docs/integrations/ide/vscode.rst

```diff
@@ -26,3 +26,43 @@
 
 #. The remote host will now be available in the VS Code **Remote Explorer**. For further detail,
    please refer to the `official documentation <https://code.visualstudio.com/docs/remote/ssh>`__.
+
+############################
+ Addendum for Windows Users
+############################
+
+#. On Windows, VSCode uses CMD to run its ssh command under the hood, so we have to modify a few
+   things in this config file. First, prepend the proxy command to use WSL. In a CMD shell, type
+   `where wsl` and prepend the result to the ProxyCommand entry in your config file. This should
+   lead to a ProxyCommand that looks like this:
+
```

Comment:
#. On Windows, Visual Studio Code uses CMD to run its SSH command under the hood. Therefore, we need to make a few modifications to the config file. First, prepend the proxy command to use WSL. In a CMD shell, type
   ``where wsl`` and prepend the result to the ProxyCommand entry in your config file. This should
   lead to a ProxyCommand that looks like this:

Reply:
Updated